pkg/blobstore/configuration/new_blob_access.go

```diff
@@ -239,9 +240,15 @@ func newNestedBlobAccessBare(configuration *pb.BlobAccessConfiguration, creator
 					DialTimeout:     dialTimeout,
```

Comment:
Now that we're switching to a version of go-redis that has support for context, is there still any value in providing config options for DialTimeout, ReadTimeout and WriteTimeout?

In an ideal world, all of Buildbarn's components should continue to go on, as long as Bazel on the client side is willing to do so.

Reply:
This is an excellent question which I am currently in the middle of learning the answer to, because something in here is misbehaving. I'll probably know the answer in the next day or two.

Reply:
So the answer turns out to be that these are the settings for net.Dialer, mirroring those names, but somewhat misleadingly. They're the timeouts for a single attempt at a dial/read/write, which are inside the retry loop. MaxRetries * DialTimeout needs to be significantly smaller than the context deadline, so they are both useful and important.

I shall document them better.

Reply:
Documented in the .proto accordingly. They turn out to be important in the clustered configuration, because the defaults are way too long to be hanging on a single node when you could be trying a different one.